DFE-Digital/teacher-services-cloud
# [426] Create managed identity for the cluster

## What
Split from: https://github.com/DFE-Digital/teacher-services-cloud/pull/77

The cluster must have an identity with permission on the resource group where the public IPs are deployed. We create it and request the new permissions. Then we will be able to make the public IPs move.

## How to review
- Deploy azure resources
- Deploy cluster,
- It should have a new managed identity. you can check with `az aks show -n s189d01-tsc-cluster1-aks -g s189d01-tsc-dv-rg`
